src/biogeochem/ch4Mod.F90

```diff
@@ -3941,7 +3941,7 @@ logical                , intent(in)    :: lake      ! function called with lake
 
          errch4(c) = errch4(c) + (ch4_surf_aere(c) + ch4_surf_ebul(c) + ch4_surf_diff(c))*dtime
 
-         if (abs(errch4(c)) < 1.e-8_r8) then
```

Comment:
Can this error check go back to what it was?

Reply:
I didn't notice this change before. We need further test.

Reply:
I have changed it back